### DISCUSSION TITLE
Not able to use OTEL_INSTRUMENTATION_METHODS_INCLUDE to create separate spans for methods

### DISCUSSION THREAD

**Discussion by mvkrishna86** [open-telemetry/opentelemetry-java-instrumentation#5422] [Q&A] [answered]
Hi,
We are using auto instrumentation for our springboot services. And we are having some requirement to trace specific methods in the code. I tried by adding this environment variable (OTEL_INSTRUMENTATION_METHODS_INCLUDE) with the classes and methods that we want to trace. But I dont see any spans created for those methods. Can someone help me here?

```
OTEL_EXPORTER_OTLP_HEADERS=x-scope-orgid=XXXXXXXXXX
OTEL_METRICS_EXPORTER=prometheus
OTEL_INSTRUMENTATION_METHODS_INCLUDE=com.pkg.Class1[method1],com.pkg.Class2[method2]
OTEL_RESOURCE_ATTRIBUTES=service.name=SERVICE-NAME
OTEL_TRACES_EXPORTER=otlp
OTEL_PROPAGATORS=b3multi
OTEL_EXPORTER_OTLP_ENDPOINT=http://grafana-agent:4317
OTEL_INSTRUMENTATION_COMMON_DB_STATEMENT_SANITIZER_ENABLED=false
```

**Comment by trask** [comment, marked as answer]
hi @mvkrishna86! I think that should be a semicolon `;` instead of a comma `,` e.g.

`OTEL_INSTRUMENTATION_METHODS_INCLUDE=com.pkg.Class1[method1];com.pkg.Class2[method2]`

**Reply by mvkrishna86** [reply]
oh.. thanks for correcting. I will check and update you :)
